### DISCUSSION TITLE
如何关闭接口文档的 Schemas

### DISCUSSION THREAD

**Discussion by zizle** [fastapi/fastapi#6758] [Questions] [answered]
如题，我想关闭接口文档页面里的 Schemas，应该怎么做呢？求解惑

**Comment by waynerv** [comment, marked as answer]
Check out this issue https://github.com/tiangolo/fastapi/issues/2633#issuecomment-759017346.

**Comment by zizle** [comment]
我将静态文件js，和css本地化了。重写了文档的API，然后通过本地css文件将其隐藏了。

**Comment by tiangolo** [comment]
Thanks for the help @waynerv! :bow: 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
